src/ApprenticeAan/SFA.DAS.ApprenticeAan.Application/CalendarEvents/Queries/GetCalendarEventById/GetCalendarEventByIdQueryResponse.cs

```diff
@@ -0,0 +1,6 @@
+﻿using SFA.DAS.ApprenticeAan.Application.CalendarEvents.Queries.GetCalendarEvents;
+using System.Net;
+
+namespace SFA.DAS.ApprenticeAan.Application.CalendarEvents.Queries.GetCalendarEventById;
+
+public record GetCalendarEventByIdQueryResponse(CalendarEvent? CalendarEvent, HttpStatusCode StatusCode);
```

Comment:
This is redundant and should be remvoed

Reply:
I'm using it to mock the `HttpStatusCode` since I can't mock the `Response`